moveit_ros/planning_interface/package.xml

```diff
@@ -8,34 +10,58 @@
   <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
   <maintainer email="me@v4hn.de">Michael Görner</maintainer>
   <maintainer email="moveit_releasers@googlegroups.com">MoveIt! Release Team</maintainer>
+  <maintainer email="anasarrak@erlerobotics.com">Anas M'chichou</maintainer>
 
   <license>BSD</license>
 
   <url type="website">http://moveit.ros.org</url>
   <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
   <url type="repository">https://github.com/ros-planning/moveit</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <buildtool_depend>python-catkin-pkg</buildtool_depend>
-
-  <depend>moveit_ros_planning</depend>
-  <depend>moveit_ros_warehouse</depend>
-  <depend>moveit_ros_move_group</depend>
-  <depend>moveit_ros_manipulation</depend>
-  <depend>roscpp</depend>
-  <depend>rospy</depend>
-  <depend>rosconsole</depend>
-  <depend>actionlib</depend>
-  <depend>geometry_msgs</depend>
-  <depend>moveit_msgs</depend>
-  <depend>tf2</depend>
-  <depend>tf2_eigen</depend>
-  <depend>tf2_geometry_msgs</depend>
-  <depend>tf2_ros</depend>
-  <depend>python</depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>moveit_ros_planning</build_depend>
+  <build_depend>moveit_ros_warehouse</build_depend>
+  <build_depend>moveit_ros_move_group</build_depend>
+  <build_depend>moveit_ros_manipulation</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>rosconsole</build_depend>
+  <build_depend>actionlib</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>moveit_msgs</build_depend>
+  <build_depend>tf2</build_depend>
+  <build_depend>tf2_eigen</build_depend>
+  <build_depend>tf2_geometry_msgs</build_depend>
+  <build_depend>tf2_ros</build_depend>
+  <build_depend>python</build_depend>
   <build_depend>eigen</build_depend>
 
+  <exec_depend>moveit_ros_planning</exec_depend>
+  <exec_depend>moveit_ros_warehouse</exec_depend>
+  <exec_depend>moveit_ros_move_group</exec_depend>
+  <exec_depend>moveit_ros_manipulation</exec_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>rospy</exec_depend>
+  <exec_depend>rosconsole</exec_depend>
+  <exec_depend>actionlib</exec_depend>
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>moveit_msgs</exec_depend>
+  <exec_depend>tf2</exec_depend>
+  <exec_depend>tf2_eigen</exec_depend>
+  <exec_depend>tf2_geometry_msgs</exec_depend>
+  <exec_depend>tf2_ros</exec_depend>
+  <exec_depend>python</exec_depend>
+  <exec_depend>eigen</exec_depend>
```

Comment:
is there any package in ROS 2.0 called `python` or `eigen`. Otherwise you need to remove these lines

Reply:
@anasarrak can you please address these concerns?

Reply:
Changes already made https://github.com/AcutronicRobotics/moveit2/pull/23/commits/34d992b346010d05a37b8158c503decaaa5e238e